west.yml

```
@@ -72,7 +72,7 @@ manifest:
       revision: b52fdbf4b62439be9fab9bb4bae9690a42d2fb14
       path: modules/hal/st
     - name: hal_stm32
-      revision: 68bfdabe97aa60934e1519e98593abaa9745501e
+      revision: 4cc96d4106a5204e3af360dfa6449c120332969f
```

Reference should be as follows
```suggestion
      revision: pull/92/head
```
This allows to always be using the latest module branch version from zephyr repo

---

Thanks for the hint, I wasn't aware of this feature. I have just fixed that.